index.html

```diff
@@ -2,71 +2,268 @@
-              class="social__icon"
-                src="./images/social/linkedin.svg"
-                alt="linkedin"
+            <a href="#" title="social links"
+              ><img src="./images/social/linkedin.svg" alt="linkedin"
             /></a>
           </li>
           <li>
-            <a href="#" title="social links" class="social__link"
-              ><img
-              class="social__icon"
-                src="./images/social/medium.svg"
-                alt="medium"
+            <a href="#" title="social links"
+              ><img src="./images/social/vector.svg" alt="vector"
             /></a>
           </li>
           <li>
-            <a href="#" title="social links" class="social__link"
-              ><img
-              class="social__icon"
-                src="./images/social/github.svg"
-                alt="github"
+            <a href="#" title="social links"
+              ><img src="./images/social/twitter.svg" alt="twitter"
             /></a>
           </li>
           <li>
-            <a href="#" title="social links" class="social__link"
-              ><img
-              class="social__icon"
-                src="./images/social/vector.svg"
-                alt="vector"
+            <a href="#" title="social links"
+              ><img src="./images/social/medium.svg" alt="medium"
             /></a>
           </li>
         </ul>
+        <button class="card__button card__button--about" type="button">
+          Get my resume
+        </button>
+        <ul class="skills">
+          <li>
+            <details open>
+              <summary class="flex stack">
+                <h4 class="stack__item">Languages</h4>
+                <img class="arrow" src="./images/arrow.svg" alt="" />
+              </summary>
+              <ul class="lang flex flx-dir-col">
+                <li class="lang__items grid">
+                  <img
+                    src="./images/languages/javascript.svg"
+                    alt="javaScript"
+                  />
+                  <h4>JavaSript</h4>
+                </li>
+                <li class="lang__items grid">
+                  <img src="./images/languages/html.svg" alt="html" />
+                  <h4>HTML</h4>
+                </li>
+                <li class="lang__items grid">
+                  <img src="./images/languages/css.svg" alt="css" />
+                  <h4>CSS</h4>
+                </li>
+              </ul>
+            </details>
+            <details>
+              <summary class="flex border">
+                <h3 class="">Frameworks</h3>
+                <img src="./images/arrow.svg" alt="arrow icon" />
+              </summary>
+            </details>
+            <details>
+              
+              <summary class="flex border">
+                <h3 class="">Skills</h3>
+                <img src="./images/arrow.svg" alt="arrow icon" />
+              </summary>
+            </details>
+          </li>
+        </ul>
```

Comment:
- [ ] I would suggest you please stick to the design as much as possible like color, font, spacing etc. I notice that your design as figma design is almost similar but there is small changes you need to make. Please see the image below and adjust the spacing.
### Your design
![his](https://github.com/IsaMarvin/Portfolio_Microverse/assets/81727988/c419ccff-6bbc-42d8-abdc-11d56b4f113b)

#### Figma design
![fimga](https://github.com/IsaMarvin/Portfolio_Microverse/assets/81727988/89b23ca1-7397-44c6-8eae-b8ac9a4644ba)